the_mines/get_html.py

```diff
@@ -0,0 +1,47 @@
+from argparse import ArgumentParser
+import requests
+import shutil
+
+
+def get_html():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--matchday", action="store", required=True, help="ex. 8",
+    )
+    parser.add_argument(
+        "--season", action="store", required=True, help="ex. 2019/2020",
+    )
+    args = parser.parse_args()
+
+    matchday = f"{args.matchday}"
+    season = f"{args.season}"
```

Comment:
I don't know if I understand the purpose of these lines.  Is this a way of casting the `args` to `str`?

Reply:
I had to look up how to access multiple variables in an `args` object, so this is following something I found on stackoverflow. How would you do it?